Common.md

```diff
@@ -20,3 +42,135 @@ Language names are title cased.
 | [dropwizard](dropwizard) | `DropWizard` | `NewRelic-DropWizard-Exporter/0.1.0` |
 
 A full `User-Agent` example might look like `NewRelic-Python-TelemetrySDK/0.2.3 NewRelic-Python-OpenCensus/0.1.0`.
+
+## Attributes
+
+The origins of measurements exported to New Relic needs to be identifiable in order for them to be useful.
+Identify what generated, measured, collected, and transmitted measurements to New Relic is essential in providing the curated experience customers expect.
+This section describes the attributes that are used to serve this purpose.
+
+The following table is an overview for quick reference of all the attributes defined in the following sub-sections.
+
+| Attribute | REQUIRED/RECOMMENDED/OPTIONAL |
+| --------- | ----------------------------- |
+| `instrumentation.provider` | REQUIRED |
+| `instrumentation.name` | RECOMMENDED |
+| `instrumentation.version` | RECOMMENDED |
+| `collector.name` | RECOMMENDED |
+| `collector.version` | RECOMMENDED |
+| `host.hostname` | REQUIRED† |
+| `service.name` | REQUIRED† |
+| `app.name` | REQUIRED† |
+| `mobileApp.name` | REQUIRED† |
+| `device.model` | RECOMMENDED† |
+| `browserApp.name` | REQUIRED† |
+| `db.name` | REQUIRED† |
+
+†: applies only if annotating an applicable source.
+See the related sub-section for more information.
+
+### Instrumentation
```

Comment:
This section doesn't seem quite right to me. I think of instrumentation as the *usage* of an instrumentation API. So, for example, micrometer provides an API to build instrumentation. Spring-boot has instrumentation that is built using the micrometer metric APIs. 

That's what I have in my brain, though. Perhaps my understanding is not the common one?

Reply:
Updated. LMK if closer.

Reply:
looks great.